recipes/pydeck-earthengine-layers/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pydeck-earthengine-layers" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "0cf5c8034a44f2ecd7e925e5033f853ecb420ef4ddcf49dc4f4f8d6ff736e96b"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - earthengine-api
+    - jinja2
+    - pip
+    - pydeck
+    - python
+    - requests
+    - setuptools
```

Comment:
you don't need pip and setuptools.  Also, do you need all of these packages to build, or just at run time?

Reply:
It's a pure-python package, so I'm guessing I don't need these to build

Reply:
@scopatz But as you can see, I was getting failing builds without them

Reply:
Weird ok... I guess you do

Reply:
I don't know Conda forge packaging very well, so not sure why. But in any case, I reverted the last commit for now.